en/change-log.md

```diff
@@ -7,7 +7,24 @@ _keywords: App Builder, Web App Builder, Design Systems, Design Systems UX, UI k
 
 > [!NOTE]
 
-><b>Latest update was made on August 16th, 2022.
+><b>Latest update was made on 11th of October, 2022.
+
```

Comment:
### Features
- Added **Tree-Grid** and **Accordion** to the component toolbox
- Define columns collection for **Grid** component, and re-arrange columns
- Added  new **sample app**- HR Dashboard

### Improvements
- Added support for reference objects when using **OpenAPI** definition
- Added support for **Safari browser**
- **Generate app** with licensed packages if you own a subscription 
- Angular apps are generated using **Ignite UI for Angular `~14.1.0`** [Change log](https://github.com/IgniteUI/igniteui-angular/blob/master/CHANGELOG.md#1410)
- Bug fixes and general improvements

[LEARN MORE](https://www.infragistics.com/community/blogs/b/infragistics/posts/app-builder-release-with-web-components-code-generation-tree-grid-and-more)